javascript-carousel/index.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>JavaScript Carousel</title>
+  <link
+    rel="stylesheet"
+    href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.3/css/all.min.css">
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <div class="container">
+    <div class="row">
+      <div class="column-side">
+        <i class="fas fa-chevron-left previous"></i>
+      </div>
+      <img src="images/001.png" alt="pokemon" class="image">
+      <div class="column-side">
+       <i class="fas fa-chevron-right next"></i>
+      </div>
+    </div>
+  </div>
+  <div class="container hidden">
+    <div class="row">
+      <div class="column-side">
+        <i class="fas fa-chevron-left previous"></i>
+      </div>
+      <img src="images/004.png" alt="pokemon" class="image">
+      <div class="column-side">
+        <i class="fas fa-chevron-right next"></i>
+      </div>
+    </div>
+  </div>
+  <div class="container hidden">
+    <div class="row">
+      <div class="column-side">
+        <i class="fas fa-chevron-left previous"></i>
+      </div>
+      <img src="images/007.png" alt="pokemon" class="image">
+      <div class="column-side">
+        <i class="fas fa-chevron-right next"></i>
+      </div>
+    </div>
+  </div>
+  <div class="container hidden">
+    <div class="row">
+      <div class="column-side">
+        <i class="fas fa-chevron-left previous"></i>
+      </div>
+      <img src="images/025.png" alt="pokemon" class="image">
+      <div class="column-side">
+        <i class="fas fa-chevron-right next"></i>
+      </div>
+    </div>
+  </div>
+  <div class="container hidden">
+    <div class="row">
+      <div class="column-side">
+        <i class="fas fa-chevron-left previous"></i>
+      </div>
+      <img src="images/039.png" alt="pokemon" class="image">
+      <div class="column-side">
+        <i class="fas fa-chevron-right next"></i>
+      </div>
+    </div>
+  </div>
```

Comment:
Hmm, you shouldn't have multiple sets of chevrons like this. I would make it so that there is one set of chevrons and just multiples of the `img` elements